.github/workflows/verifiers.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       - run: python3 scripts/latex_collector.py set.mm list-set.tex
       - run: pdflatex -halt-on-error list-set.tex 2>&1 | tee ,latex-output
       # Complain if the pdflatex output says something is invalid
-      - run: ! grep ' invalid ' ,latex-output
+      - run: "! grep ' invalid ' ,latex-output"
```

Comment:
Shouldn't it be `- run: ! grep " invalid " latex-output` instead?  Following usual command line syntax and the pattern in line 142

Reply:
Once we have the build passing, I'm all for spending more time with https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell or whatever will help us gain full understanding.

I'm going to merge this now, however. Feel free to open an issue or new pull request or any other means, I just think we need to do either this or #3134 while we have that discussion.